l2.py

```diff
@@ -0,0 +1,234 @@
+
+    if not os.path.exists(cfg.chaindata_dir):
+        log_file = "logs/init_l2_genesis.log"
+        print(f"Directory {cfg.chaindata_dir} missing, importing genesis in op-geth node."
+              f"Logging to {log_file}")
+        lib.run(
+            "initializing genesis",
+            ["op-geth",
+             f"--verbosity={cfg.verbosity}",
+             "init",
+             f"--datadir={cfg.data_dir}",
+             paths.l2_genesis_path])
+
+    log_file_path = "logs/l2_node.log"
+    print(f"Starting op-geth node. Logging to {log_file_path}")
+    sys.stdout.flush()
+
+    log_file = open(log_file_path, "w")
+
+    PROCESS_MGR.start(
+        "starting op-geth",
+        [
+            "op-geth",
+
+            f"--datadir={cfg.data_dir}",
+            f"--verbosity={cfg.verbosity}",
+
+            f"--networkid={cfg.chain_id}",
+            "--syncmode=full",  # doesn't matter, it's only us
+            "--gcmode=archive",
+
+            # No peers: the blockchain is only this node
+            "--nodiscover",
+            "--maxpeers=1",
+
+            # HTTP JSON-RPC server config
+            "--http",
+            "--http.corsdomain=*",
+            "--http.vhosts=*",
+            "--http.addr=0.0.0.0",
+            f"--http.port={cfg.rpc_port}",
+            "--http.api=web3,debug,eth,txpool,net,engine",
+
+            # WebSocket JSON-RPC server config
+            "--ws",
+            "--ws.addr=0.0.0.0",
+            f"--ws.port={cfg.ws_port}",
+            "--ws.origins=*",
+            "--ws.api=debug,eth,txpool,net,engine",
+
+            # Configuration for clique signing, clique itself is enabled via the genesis file
+            f"--unlock={cfg.signer_address}",
+            "--mine",
+            f"--miner.etherbase={cfg.signer_address}",
+            f"--password={cfg.data_dir}/password",
+            "--allow-insecure-unlock",
+
+            # Authenticated RPC config
+            "--authrpc.addr=0.0.0.0",
+            "--authrpc.port=9551",
+            "--authrpc.vhosts=*",
+            f"--authrpc.jwtsecret={cfg.jwt_secret_path}",
+
+            # Configuration for the metrics server (we currently don't use this)
+            "--metrics",
+            "--metrics.addr=0.0.0.0",
+            "--metrics.port=9060",
+
+            # Configuration for the discovery server, avoid conflicts with L1 node
+            "--discovery.port=30313",
+            "--port=30313"
+        ], forward="fd", stdout=log_file, stderr=subprocess.STDOUT)
+
+    lib.wait_for_rpc_server("127.0.0.1", cfg.rpc_port)
+
+
+####################################################################################################
+
+
+def clean(paths: OPPaths):
+    """
+    Cleans up build outputs, such that trying to deploy the devnet op-geth node will proceed as though it
+    was the first invocation.
+    """
+    if os.path.exists(paths.devnet_gen_dir):
+        print(f"Cleaning up {paths.devnet_gen_dir}")
```

Comment:
Same comment as in other file, maybe make debug statement?

Reply:
not get you here.

Reply:
I mean same as https://github.com/0xFableOrg/simple-op-stack-rollup/pull/27#discussion_r1304163295 (remove prints or turn them into `lib.debug`